docker/Dockerfile.backend

```diff
@@ -25,7 +25,7 @@ COPY backend $path/backend
 FROM base AS debug
 RUN pip install --no-cache-dir debugpy watchdog
 USER backend-user
-CMD watchmedo auto-restart --recursive --pattern="*.py" --ignore-patterns="*.pyc;*__pycache__*" -- python -m debugpy --listen 0.0.0.0:${RPCDEBUGPORT} -m flask run -h 0.0.0.0 -p ${FLASK_SERVER_PORT}
+CMD watchmedo auto-restart --no-restart-on-command-exit --recursive --pattern="*.py" --ignore-patterns="*.pyc;*__pycache__*" -- python -m debugpy --listen 0.0.0.0:${RPCDEBUGPORT} -m flask run -h 0.0.0.0 -p ${FLASK_SERVER_PORT}
```

Comment:
extra: doesn't restart when code exits, which most of the time is when there are "compile" errors